packages/desmos-std/src/profiles/models_app_links.rs

```diff
@@ -0,0 +1,51 @@
+use cosmwasm_std::Addr;
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
+#[serde(rename_all = "snake_case")]
+pub struct ApplicationLink {
+    pub user: Addr,
+    pub data: Data,
+    pub state: String,
+    pub oracle_request: OracleRequest,
+    pub result: AppLinkResult,
+    pub creation_time: String,
+}
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
+#[serde(rename_all = "snake_case")]
+pub struct Data {
+    pub application: String,
+    pub username: String,
+}
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
+#[serde(rename_all = "snake_case")]
+pub struct TimeoutHeight {
+    pub revision_number: u64,
```

Comment:
After applying codec in the desmos side, we can replace all `u64` into `Uint64` safely.